README.md

```diff
@@ -41,33 +41,124 @@ Usually, operators use `ServiceBinding` resource name (`.metadata.name`) as the
 
 The specification mandates the `type` field and recommends `provider` field in the projected binding.  In many cases the `type` field should be good enough to select the appropriate binding.  In cases where it is not (e.g. when there are different providers for the same Provisioned Service), the `provider` field may be used.  For example, when the type is `mysql`, the `provider` value might be `mariadb`, `oracle`, `bitnami`, `aws-rds`, etc.  When the workload is selecting a binding, if necessary, it could consider `type` and `provider` as a composite key to avoid ambiguity.  This could be helpful if a workload needs to choose a particular provider based on the deployment environment.  In the deployment environment (`stage`, `prod`, etc.), at any given time, you need to ensure only one binding projection exists for a given `type` or `type` and `provider` -- unless your workload needs to connect to all the services.
 
-### Programming language specific library APIs
+### Environment Variables
 
-A workload can retrieve bindings through a library available for your programming language of choice.  Language-specific APIs are encouraged to follow the pattern described here, but may not.  Consult your library API documentation to confirm its behavior.
+The specification also has support for projecting binding values as environment variables.  You can use the built-in language feature of your programming language of choice to read environment variables.  The container must restart to update the values of environment variables if there is a change in the binding.
```

Comment:
Distinguish between a container restart due to a binding change vs needing to restart to pick up an updated value

```suggestion
The specification also has support for projecting binding values as environment variables.  You can use the built-in language feature of your programming language of choice to read environment variables.  The container must restart to read updated environment variable values.
```